indico/modules/events/templates/admin/event_keywords.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout/admin_page.html' %}
+{% from 'forms/_form.html' import simple_form %}
+
+{% block title %}
+    {% trans %}Allowed event keywords{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {% trans %}Define the list of allowed event keywords. An empty list implies any keyword is valid.
+    {% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {{ simple_form(form, back_button=false, disabled_until_change=False) }}
+
+    <script language="javascript">
```

Comment:
```suggestion
    <script>
```

`language` is heavily outdated

Reply:
I would not use jQuery in any new code

Reply:
How'd you do it then? with raw `document` inspection?

Reply:
Yep. It's very similar nowadays - the DOM has most of the nice things for which you needed jQuery before.

```html
<script>
    domReady.then(() => {
        const id = 'keyword-container';
        const input = document.querySelector('#{{ form.keywords.id }}');
        const field = input.parentElement;
        field.id = id;
        field.classList.add('i-taglist-field');
        input.remove();
        new Taggle(...);
    });
</script>
```

Note: To get access to the domReady promise, you need to make this change:

```diff
diff --git a/indico/web/client/js/utils/domstate.js b/indico/web/client/js/utils/domstate.js
index be9e996776..9b261a8088 100644
--- a/indico/web/client/js/utils/domstate.js
+++ b/indico/web/client/js/utils/domstate.js
@@ -12,3 +12,5 @@ export const domReady = new Promise(resolve => {
     window.addEventListener('DOMContentLoaded', resolve);
   }
 });
+
+window.domReady = domReady;
```

That way you don't have to add `window.addEventListener('DOMContentLoaded', ...);` to call it. But in case domReady isn't yet available by the time your script runs (I'd be very surprised!), just use DOMContentLoaded directly.

Reply:
Just changed it, and it works like a charm!